digitaltwin_client/samples/digitaltwin_sample_device_info/ops/deviceinfo_ops_sample.c

```diff
@@ -1,75 +1,77 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <stdbool.h>
-#include "azure_c_shared_utility/xlogging.h"
 #include "../digitaltwin_sample_device_info.h"
 #include "deviceinfo_ops.h"
 
-// Sample values
-static const char digitaltwinSample_SoftwareVersionData[] = "software-revision-1.0.2A";
-static const char digitaltwinSample_ManufacturerData[] = "Sample Manufacturer for Azure IoT C SDK enabled device";
-static const char digitaltwinSample_ModelData[] = "Sample Model 123";
-static const char digitaltwinSample_OsNameData[] = "sample-OperatingSystem-name";
-static const char digitaltwinSample_ProcessorArchitectureData[] = "Contoso-Arch-64bit";
-static const char digitaltwinSample_ProcessorManufacturerData[] = "Processor Manufacturer(TM)";
-static const int  digitaltwinSample_TotalStorageData = 83886080; // 80GB
-static const int  digitaltwinSample_TotalMemoryData = 32768; // 32 MB
+#ifndef DIGITALTWIN_SAMPLE_SOFTWARE_VERSION
+#define DIGITALTWIN_SAMPLE_SOFTWARE_VERSION "1.0.0.0"
+#endif
+
+#ifndef DIGITALTWIN_SAMPLE_MANUFACTURER
+#define DIGITALTWIN_SAMPLE_MANUFACTURER "Sample Manufacturer for Azure IoT C SDK enabled device"
+#endif
+
+#ifndef DIGITALTWIN_SAMPLE_MODEL
+#define DIGITALTWIN_SAMPLE_MODEL "Sample Model 123"
```

Comment:
We also need model to follow these rules: 1-64 chars, alphanumeric, dot and dash.

EG: "Sample-Model-123"